### ISSUE TITLE
[Code] Questions on the loss details 

### ISSUE THREAD

**Issue by jhcho99** [microsoft/MeshTransformer#52]
Hello! 
Thanks for the great work :)

I have two questions on the loss details in ```run_metro_bodymesh.py```.

1. According to the paper, L1 loss is used to minimize the error between the predicted joints and ground-truth joints. But, torch.nn.MSELoss() is used in ```run_metro_bodymesh.py``` instead of L1 loss. Why do you use MSE loss instead of L1 loss described in the paper?
![image](https://user-images.githubusercontent.com/55849968/148637730-7ec3bbe1-7352-4075-8f61-6a5787c9f42c.png)

2. Why do you subtract the location of pelvis twice?  During training, line 207-211 in ```run_metro_bodymesh.py``` computes the subtraction of the pelvis location from the given annotation
![image](https://user-images.githubusercontent.com/55849968/148637405-0b36219e-c426-4a27-87ca-e76d23c3cc2d.png)
, but line 125-128 in  ```run_metro_bodymesh.py``` also computes the subtraction of the pelvis location from the modified annotation obtained by the computation in line 207-211.
![image](https://user-images.githubusercontent.com/55849968/148637468-485566c9-42da-49e6-9d6a-042e69670585.png)

Could you please check the above questions?
Thanks!

**Comment by kevinlin311tw** [comment]
Thanks for pointing out the questions!

Q&A1: Sorry for the inconsistent descriptions in the paper/code. When we implement the loss functions, we mainly adopt the loss functions from GraphCMR (CVPR 2019). As the authors (CVPR 2019) discussed in their paper, they empirically found L1 loss gives more stable training compared to MSE loss. However, in their implementation, they actually use MSE loss for the 2D/3D joints, and L1 loss for the 3D vertices. Probably replacing MSE with L1 could further improve our training, but we haven't tried it.

Q&A2: In line 207-211, we are trying to prepare GT data, and we normalize 3D GT joints based on a pre-defined pelvis. When we compute loss in line 125-128, we want to make sure both prediction and ground truth are in the same 3D space, so we normalize them again based on their pelvis (which is computed on the fly by (left_hip+right_ hip)/2).

**Comment by jhcho99** [comment]
Thanks for the detailed explanation about my questions!
I greatly appreciate your help :)
